jplag/src/main/java/de/jplag/SubmissionSetBuilder.java

```diff
@@ -51,19 +51,26 @@ public SubmissionSetBuilder(Language language, JPlagOptions options, ErrorCollec
      * @throws ExitException if the directory cannot be read.
      */
     public SubmissionSet buildSubmissionSet() throws ExitException {
-        Set<File> rootDirectoryNames = verifyRootDirectories(options.getRootDirectoryNames());
+        Set<File> plagiarismCheckRootDirectoryNames = verifyRootDirectories(options.getPlagiarismCheckRootDirectoryNames(), true);
+        Set<File> priorSubmissionrootDirectoryNames = verifyRootDirectories(options.getPriorSubmissionsDirectoryNames(), false);
```

Comment:
The naming of the sets and the options are a bit confusing. Why not something like `submissionDirectoryNames` and `oldSubmissionDirectoryNames`? Or maybe something even better?